src/Illuminate/Process/PendingProcess.php

```diff
@@ -258,6 +258,7 @@ public function run(array|string|null $command = null, ?callable $output = null)
 
             return new ProcessResult(tap($process)->run($output));
         } catch (SymfonyTimeoutException $e) {
+            /** @phpstan-ignore variable.undefined */
```

Comment:
Ignoring errors doesn't seem right.

Maybe move `$process = $this->toSymfonyProcess($command);` out of the `try` section instead?

Reply:
Indeed. Fixed in 7ce889a.